### ISSUE TITLE
Update a11y views so that they can run on phet-server

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/chipper#1143]
From recent discussions, it is likely we will want to share the a11y view with teachers. The a11y views are packaged with the build and available with the latest version of published sims. But the URL in the a11y view is incorrect for phet-server. It includes the brand information which is removed for sims on phet-server.

I am going to update the URL so that the a11y view works on phet-dev, unbuilt, and on phet-server. We may want to do some things to verify quality of the a11y-view before sharing with teachers, but moving forward they could be used on phet-server.

If they need to be available right away we could do a maintenance release to make them usable.

**Comment by jessegreenberg** [comment]
I verified that https://github.com/phetsims/chipper/commit/51f71aeb9aabefa7463e90c10dfb31e7dec62f70 has the a11y view working with unbuilt sim and on phet-dev. I am not sure how to verify that this works with a published sim, but it is using the same pattern as sim-iframe.html.

**Comment by jessegreenberg** [comment]
From slack discussion we do not want to do a maintenance release for this now. I went to a published a11y view on phet-server and verified that 
```
  const filenameEnding = ( window.location.hostname === 'phet.colorado.edu' || !IS_BUILT ) ? '.html' : '_phet.html';
```

returns the correct suffix with the dev tools.  I think that is enough verification for now and if we decide to do a maintenance release we can come back to this issue.
